lib/active_job/queue_adapters/solid_queue_ext/workers.rb

```diff
@@ -20,8 +20,29 @@ def find_worker(worker_id)
   end
 
   private
-    def solid_queue_processes_from_workers_relation(relation)
-      SolidQueue::Process.where(kind: "Worker").offset(relation.offset_value).limit(relation.limit_value)
+
+    def workers(workers_relation)
+      SolidQueue::Process.where(kind: "Worker")
+        .then { |workers| filter_by_hostname(workers, workers_relation.hostname) }
+        .then { |workers| filter_by_name(workers, workers_relation.name) }
+        .then { |workers| limit(workers, workers_relation.limit_value) }
+        .then { |workers| offset(workers, workers_relation.offset_value) }
+    end
+
+    def filter_by_hostname(workers, hostname)
+      hostname.present? ? workers.where(hostname: hostname) : workers
+    end
+
+    def filter_by_name(workers, name)
+      name.present? ? workers.where(pid: name) : workers
```

Comment:
Hmm... does this work? Why is this used as the name?

Reply:
I guess it would make more sense to call the filter pid if we wanted to have it

Reply:
Oh, yes! +1 to `pid` 😊